ClosedXML_Examples/ConditionalFormatting/ConditionalFormatting.cs

```diff
@@ -527,6 +527,26 @@ public void Create(String filePath)
         }
     }
 
+    public class CFDataBarNegative : IXLExample
+    {
+        public void Create(String filePath)
+        {
+            var workbook = new XLWorkbook();
+            var ws = workbook.AddWorksheet("Sheet1");
+
+            ws.FirstCell().SetValue(1)
```

Comment:
One of these values should probably be negative to illustrate this use case, right?

Reply:
Yes, I change to show with negative colors